roomba/roomba.py

```diff
@@ -769,7 +769,7 @@ def interface_serial(self):
             try:
                 if self.qsens.qsize > 0:
                     self.qsens.get_nowait()
-            except:
+            except:  # noqa: E722
```

Comment:
I think this can be excluded globally from the setup.cfg file. What is the humanly readable error type associated to it?

Reply:
OK. I'll add 

# E722 do not use bare except
ignore = E722

 in flake8 config